src/components/Grants/GrantCard/ProfileGrantItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Link } from 'decentraland-gatsby/dist/plugins/intl'
+import { Back } from 'decentraland-ui/dist/components/Back/Back'
+
+import { GrantAttributes } from '../../../entities/Proposal/types'
+import { isProposalInCliffPeriod } from '../../../entities/Proposal/utils'
+import locations from '../../../modules/locations'
+import { formatDate } from '../../../modules/time'
+import Username from '../../User/Username'
+import GrantPill from '../GrantPill'
+
+import CliffProgress from './CliffProgress'
+import './ProfileGrantItem.css'
+import ProgressBarTooltip from './ProgressBarTooltip'
+import VestingProgress from './VestingProgress'
+
+interface Props {
+  grant: GrantAttributes
+}
+
+function ProfileGrantItem({ grant }: Props) {
+  const t = useFormatMessage()
+  const intl = useIntl()
+  const { user, title, enacted_at, token } = grant
+  const enactedDate = new Date(enacted_at * 1000)
```

Comment:
I'd move `value => Time.unix(value).fromNow()` to a helper for now. (`helpers/grants.ts`?)

Not sure how the Grant class is going to work. Are we going to return a Grant from the Governance client?

Reply:
I was thinking on wrapping the Grants that the governance client returns somewhere in the frontend, and having a frontend only object (because methods associated with it will be related to how the object displays/shows it's info)

The advantage to having this kinds of functions scattered on util/helper files is that you don't need to remember they exist.
When you add the methods to the object, then the only way to fetch the data you need to use is through the object's interface.